src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Inject;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        return cinemaHallDao.get(id).get();
```

Comment:
The `get` method directly calls `.get()` on the `Optional` returned by the DAO, which can lead to `NoSuchElementException` if the cinema hall is not found. Consider using `orElseThrow()` to provide a more meaningful exception or handle the absence of a value more gracefully.